dashboard/app/models/pd/application/teacher1819_application.rb

```diff
@@ -788,15 +794,25 @@ def yes_no_response_to_yes_no_score(response)
 
     def able_attend_multiple_to_yes_no_score(response)
       response = response.join
-      if response.start_with?(TEXT_FIELDS[:no_explain])
+      if response.start_with?(TEXT_FIELDS[:no_explain]) || response == NO
         NO
-      elsif response && !response.include?(TEXT_FIELDS[:no_explain])
+      elsif (response && !response.include?(TEXT_FIELDS[:no_explain])) || response == YES
         YES
       else
         nil
       end
     end
 
+    def able_attend_single_to_yes_no_score(response)
+      if [TEXT_FIELDS[:able_to_attend_single], YES].include? response
+        YES
+      elsif (response && !response.include?(TEXT_FIELDS[:unable_to_attend])) || response == NO
```

Comment:
Similarly, how can this be `NO`? Won't it always be one of the 2 options: `TEXT_FIELDS[:able_to_attend_single]` or `TEXT_FIELDS[:unable_to_attend]` ?